### ISSUE TITLE
Getting characteristion data for INHERITED compounds

### ISSUE THREAD

**Issue by edwintse** [OpenSourceMalaria/OSMSeries4Paper1#14]
There are a number (108) of compounds in the Series 4 master list that have been **inherited** from the CRO when we first picked up the project. As a result, we don't actually have most (if not all) of the characterisation for these compounds.

- This is a google sheet of just the inherited compounds from the master list ([here](https://docs.google.com/spreadsheets/d/13ArIVrqSBaQr39tHSCEp_7yt8QPmyJ_UHe_gX2mfzrI/edit?usp=sharing))
- This is the pdf of the inherited compounds, including structures and potency data ([here](http://malaria.ourexperiment.org/biological_data/10041))

Since we will need this data for the paper we're writing, it would be good to get in touch with the CRO to see if they have any of the characterisation data stored somewhere.

**Comment by cdsouthan** [comment]
Agreed  and please expand/find out more context.  So did MMV contract out analogues synthesis of "our" S4s?  None of this is necessarily problematic  but (as open practitioners) we need to know and provenance who the CRO was and the explicit structural data (that MMV paid for).  So will it be down to your good self @edwintse to complete new OSM numbers?   When you have done that its not a bad idea to send these back to MMV so they can synonym-track in their own systems.

**Comment by danaklug** [comment]
Now seems like a good time to come back to this... @edwintse any updates?

**Comment by cdsouthan** [comment]
Too much water under the bridge?  Suggest we just bash on.  Referees might pick up on this  but we'd be unlucky

**Comment by edwintse** [comment]
So as far as I know, MMV contracted TGS Lifesciences to do the initial lead optimisation (before we started working on it). I've added OSM numbers to the above spreadsheet of inherited compounds. The majority of them follow OSM-X- as the code but there are a small handful that have OSM-S- codes even though they weren't made by us (not sure of the reason for this). Of all these compounds in the list, only 1 has been remade by us (MMV669784 for IVIVC). We only have the experimental procedures for the synthesis of MMV670652 but this only contains HNMR for select intermediates and LCMS for the final compound.

Essentially, we could try to contact the relevant people but I suspect we won't be getting much back.

**Comment by danaklug** [comment]
@cdsouthan @edwintse Agree it's probably not worth the effort - just wondering if there'd been any updates on this since it was last discussed. I think it should be fine without.

**Comment by cdsouthan** [comment]
Were any of these frontrunners for the paper?  i.e. 100 nM-ish?  If so prudent to remake and retest?

**Comment by edwintse** [comment]
Actually, the majority of these compounds are >100 nM. There are only 6 that are <100 nM:

For two of them (MMV669844 and MMV670437) the racemates were made and tested previously. MMV670652 has a benzylic OCHF2 but I don't think it's worth the effort to make this (synthesis is relatively complicated). There's a 6-substituted ether (MMV670945) that might be being worked on in Sydney. MMV672936 has a benzylic F. MMV670947 is a 3,4-difluorophenyl compounds with a benzylic primary OH.

I wouldn't say that any of these are essential for the paper. The most relevant would probably be MMV670947 which is reasonable to make if needed.

**Comment by mattodd** [comment]
Update on inherited compounds. I've received [a file from Abhijit Kundu at TCG who kindly provided 1H NMR and LCMS data for the compounds we need](https://github.com/OpenSourceMalaria/OSMSeries4Paper1/blob/master/Experimental/Inherited%20Compounds%20TCG/Analytical%20data_Final%20compounds_2021-03-18_AK.xls). He's clarified that an author, Saumitra Sengupta, needs to be added. I have asked the following today by way of follow up:

1) I'm assuming that it's OK to put this file in the public domain? We'll also include the data in the paper supporting info.

2) Do you have any indication of the purity of the compounds? e.g. are they all 95% pure, typically, on the LCMS trace?

3) For authorship: is it yourself and Saumitra, or just Saumitra? Is the relevant address "TCG Life Sciences, Kolkata - R&D Centre, Chemistry, Block BN, Plot 7, Salt Lake Electronics Complex, Sector V, Kolkata – 700091, West Bengal, India"? Does Saumitra have an email address? (most journals ask us for this).

4) Will TCG need to approve submission of the final manuscript?

Will let you know what I hear.

Action on @danaklug to check that these data are basically in line with what we expect.

**Comment by mattodd** [comment]
Update. 
Saumitra Sengupta has been added as an author to the paper. 
SI (NMR, MS) data uploaded - some minor queries from Ed and Dana forwarded to TCG before final incorporation into paper SI. Awaiting response.
TCG will not need to approve final submission of the manuscript.

**Comment by mattodd** [comment]
Update. In repsonse to our queries, Saumitra kindly forwarded raw data for the compounds that we could use to check any apparent disparities. @edwintse and @danaklug have gone through these and resolved (so we're all set?).

However, Ed noticed that we have since included MMV669844, MMV670652, MMV669009. Saumitra has provided the data for these too. @edwintse are these now included?

There was one final discrepancy relating to **MMV669103**. The datasheet that TCGLS provided doesn’t seem to match the structure in the original information received from MMV when OSM inherited the project. The MMV structure has a CH2 between the sulfonamide and the phenyl ring whereas TCGLS's doesn’t. The answer to this is that there is some residual uncertainty in the structure of this compound. **Shall we omit it from the paper and place a question mark in the Master List?**

Saumitra asked to be removed as an author and agreed it would be good to mention the TCGLS team in the acknowledgements. This has been done, but we should check that the relevant people are named.

**Comment by danaklug** [comment]
MMV669844, MMV670652, MMV669009 experimental has been written up.

MMV669103 has now been removed from the paper. It seems clear from the TGC data that compound **2** is the one that was made. Compound **1** is the current structure in the master list. So either the structure should be amended or a note to this discussion should be added - not sure of the preferred way to handle this. Once that has been done, this issue can be closed.

![image](https://user-images.githubusercontent.com/53487492/123616737-7afd0780-d7fe-11eb-81d8-af60f2e1c477.png)

**Comment by mattodd** [comment]
@edwintse @danaklug I think this issue can be closed **if** we have inserted a note in the Master List for compound MMV669103 that links to this discussion - some uncertainty re structure, essentially.
No more actions here for the paper re inherited compounds, correct?
